### PR TITLE
fix : create mailables class in custom mail directory

### PR DESCRIPTION
**Pull request by Bhagyrajaj**
Fix creating mail class with custom directory

## Description

While updating config file to 'mailables_dir' => app_path('Mail/Templates'), mailables are not creating in Mail/Templates directory.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/MailEclipse.php [hunk 1]
```diff
@@ -395,6 +395,10 @@ public static function getMailableTemplateData($mailableName)
     public static function generateMailable($request = null): JsonResponse
     {
         $name = self::generateClassName($request->input('name'));
+        $defaultDirectory = 'Mail';
+
+        $mailableDir = config('maileclipse.mailables_dir');
+        $customPath = substr($mailableDir,strpos($mailableDir,$defaultDirectory) + strlen($defaultDirectory) + 1);
```
Comment (ReeceM) [inline src/MailEclipse.php:401]: Hi there @Bhagyrajaj,

Would it be possible to correct this according to StyleCI please:
https://github.styleci.io/analyses/GDQpEO

**Comment by ReeceM** [comment]
Hi @Bhagyrajaj, thank you for your PR.

Just want to check, have you tested the final results work with all features of the package?

**Review by ReeceM — CHANGES_REQUESTED** [review]

File: src/MailEclipse.php (hunk 1 above)
Comment (Bhagyrajaj) [reply src/MailEclipse.php]: sure

Comment (ReeceM) [reply src/MailEclipse.php]: Thank you

**Review by ReeceM — APPROVED** [review]

**Pull request merged**